lib/Value/Matrix.pm

```diff
@@ -13,40 +13,40 @@ MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Clas
 MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
 CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
 
-Allowing Matrices in Fractions: 
-L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>	
+Allowing Matrices in Fractions:
+L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
 
      Context()->parens->set("[" => {formMatrix => 1});
 
 Files interacting with Matrices:
 
-L<lib/MatrixReal1.pm>
+L<MatrixReal1>
 
-L<macros/MatrixReduce.pl>
+L<MatrixReduce.pl>
 
-L<lib/Matrix.pm> 
+L<Matrix>
 
-L<macros/MatrixCheckers.pl> -- checking whether vectors form a basis
+L<MatrixCheckers.pl> -- checking whether vectors form a basis
 
-L<macros/MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
+L<MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
 
-L<macros/MatrixUnits.pl>   -- Generates unimodular matrices with real entries
+L<MatrixUnits.pl>   -- Generates unimodular matrices with real entries
 
-L<macros/PGmatrixmacros.pl>
+L<PGmatrixmacros.pl>
 
-L<macros/PGmorematrixmacros.pl>
+L<PGmorematrixmacros.pl>
 
-L<macros/PGnumericalmacros.pl>
+L<PGnumericalmacros.pl>
 
-L<macros/tableau.pl>
+L<tableau.pl>
 
-L<macros/quickMatrixEntry.pl>
+quickMatrixEntry.pl
```

Comment:
Is this one supposed to have the `L<...>`?

Reply:
That macro does not have POD.  So there is nothing to link to in the generated POD files.  So no.